assets/src/scss/layout/_navbar.scss

```diff
@@ -92,6 +92,10 @@ $navbar-default-height: 60px;
 }
 
 #nav-main-desktop {
+  & ~ .nav-languages {
```

Comment:
I don't understand why this is needed, since [we already hide it](https://github.com/greenpeace/planet4-master-theme/pull/1699/files#diff-2b046e4198d9cd54ff885e00329eb2495f4c793e9e83fbb43a08230f33fc4d1aR15-R17) in `languages.scss`? 🤔